src/test/ValidationFeiRiskParamsUpdate.sol

```diff
@@ -0,0 +1,122 @@
+        uint16 referralCode
+    ) external;
+}
+
+contract ValidationFeiRiskParamUpdate is Test {
+    address internal constant AAVE_WHALE =
+        0x25F2226B597E8F9514B3F68F00f494cF4f286491;
+
+    address internal constant FEI = 0x956F47F50A910163D8BF957Cf5846D573E7f87CA;
+
+    address public constant A_FEI = 0x683923dB55Fead99A79Fa01A27EeC3cB19679cC3;
+
+    ILendingPool public constant AAVE_LENDING_POOL = ILendingPool(0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9);
+
+    // can't be constant for some reason
+    string internal MARKET_NAME = "AaveV2Ethereum";
+
+    function setUp() public {}
+
+    /// @dev Uses an already deployed payload on the target network
+    function testProposalPostPayload() public {
+        /// deploy payload
+        FeiRiskParamsUpdate fei = new FeiRiskParamsUpdate();
+        _testProposal(address(fei));
+    }
+
+    function _testProposal(address payload) internal {
+        ReserveConfig[] memory allConfigsBefore = AaveV2Helpers
+            ._getReservesConfigs(false, MARKET_NAME);
+
+        address[] memory targets = new address[](1);
+        targets[0] = payload;
+        uint256[] memory values = new uint256[](1);
+        values[0] = 0;
+        string[] memory signatures = new string[](1);
+        signatures[0] = "execute()";
+        bytes[] memory calldatas = new bytes[](1);
+        calldatas[0] = "";
+        bool[] memory withDelegatecalls = new bool[](1);
+        withDelegatecalls[0] = true;
+
+        uint256 proposalId = AaveGovHelpers._createProposal(
+            vm,
+            AAVE_WHALE,
+            IAaveGov.SPropCreateParams({
+                executor: AaveGovHelpers.SHORT_EXECUTOR,
+                targets: targets,
+                values: values,
+                signatures: signatures,
+                calldatas: calldatas,
+                withDelegatecalls: withDelegatecalls,
+                ipfsHash: bytes32(0)
+            })
+        );
+
+        AaveGovHelpers._passVote(vm, AAVE_WHALE, proposalId);
+
+        ReserveConfig[] memory allConfigsAfter = AaveV2Helpers
+            ._getReservesConfigs(false, MARKET_NAME);
+
+        ReserveConfig memory feiConfigBefore = AaveV2Helpers._findReserveConfig(allConfigsBefore, "FEI", true);
+        ReserveConfig memory feiConfigAfter = AaveV2Helpers._findReserveConfig(allConfigsAfter, "FEI", true);
+
+        assertEq(feiConfigBefore.isFrozen, false);
+        assertEq(feiConfigAfter.isFrozen, true);
+        assertEq(feiConfigAfter.reserveFactor, 10_000);
+        vm.startPrank(0x4fb2e1E718EC713aa7346687FCE8e4411b7F423e);
+        uint256 balanceBefore = IERC20(FEI).balanceOf(0x4fb2e1E718EC713aa7346687FCE8e4411b7F423e);
+        AAVE_LENDING_POOL.withdraw(FEI, IERC20(A_FEI).balanceOf(0x4fb2e1E718EC713aa7346687FCE8e4411b7F423e), 0x4fb2e1E718EC713aa7346687FCE8e4411b7F423e);
+        uint256 balanceAfter = IERC20(FEI).balanceOf(0x4fb2e1E718EC713aa7346687FCE8e4411b7F423e);
+        assertEq(balanceAfter > balanceBefore, true);
```

Comment:
Can you put a space between this section and the asserts above it and also put a comment saying what exactly you're testing here: i.e `Checking if you can withdraw after freezing market`